plugins/themes/default/DefaultThemePlugin.inc.php

```diff
@@ -38,19 +31,106 @@ public function isActive() {
 	 * @return null
 	 */
 	public function init() {
-		// Load Noto Sans font from Google Font CDN
+
+		// Register theme options
+		$this->addOption('typography', 'radio', array(
+			'label' => 'plugins.themes.default.option.typography.label',
+			'description' => 'plugins.themes.default.option.typography.description',
+			'options' => array(
+				'notoSans' => 'plugins.themes.default.option.typography.notoSans',
+				'notoSerif' => 'plugins.themes.default.option.typography.notoSerif',
+				'notoSerif_notoSans' => 'plugins.themes.default.option.typography.notoSerif_notoSans',
+				'notoSans_notoSerif' => 'plugins.themes.default.option.typography.notoSans_notoSerif',
+				'lato' => 'plugins.themes.default.option.typography.lato',
+				'lora' => 'plugins.themes.default.option.typography.lora',
+				'lora_openSans' => 'plugins.themes.default.option.typography.lora_openSans',
+			)
+		));
+
+		$this->addOption('baseColor', 'color', array(
+			'label' => 'plugins.themes.default.option.color.label',
+			'description' => 'plugins.themes.default.option.color.description',
+			'default' => '#1E6292',
+		));
+
+		// Load primary stylesheet
+		$this->addStyle('stylesheet', 'styles/index.less');
+
+		// Store additional LESS variables to process based on options
+		$additionalLessVariables = '';
```

Comment:
I'll make it an array.

Technically it can accept any LESS string, but this basically takes one lump string of LESS. I want to encourage theme authors to only use this technique for variables.

If they have an entire set of LESS statements, they should put them into a file and use [this syntax](https://github.com/NateWr/default-child#modify-an-existing-style).